test/externalTests/common.sh

```diff
@@ -153,27 +153,25 @@ function force_solc
 
     printLog "Forcing solc version..."
     cat >> "$config_file" <<EOF
-module.exports['compilers'] = {solc: {version: "$dir/solc"} };
+module.exports['compilers'] = {solc: {version: "$dir/solc", settings: $(solc_settings "$OPTIMIZER_LEVEL" istanbul)}};
 EOF
 }
 
 function force_solc_settings
 {
     local config_file="$1"
     local settings="$2"
-    local evmVersion="$3"
```

Comment:
Note to self: this function now accepts two instead of three arguments

Reply:
And back to four :P
I wouldn't get too attached to the current signatures of these functions :)